src/commands/bundle.ts

```diff
@@ -0,0 +1,81 @@
+import { Flags } from '@oclif/core';
+import { Example } from '@oclif/core/lib/interfaces';
+import Command from '../base';
+import bundle from '@asyncapi/bundler';
+import fs from 'fs';
+import path from 'path';
+import { ErrorLoadingSpec } from '../errors/specification-file';
+
+export default class Bundle extends Command {
+  static description = 'Bundle multiple asyncapi files together.';
+  static strict = false;
+
+  static examples: Example[] = [
+    'asyncapi bundle ./asyncapi.yaml -o final.yaml',
+    'asyncapi bundle ./asyncapi.yaml ./spec.yaml --reference-into-components'
+  ];
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+    output: Flags.string({ char: 'o', description: 'The output file name. Omitting this flag will create a main.yaml.' }),
```

Comment:
```suggestion
    output: Flags.string({ char: 'o', description: 'The output file name. Omitting this flag will create a main.yaml.' }),
```

`main.yml` or maybe `bundle.yml`? I would also keep promoting `asyncapi` as suffix, so `asyncapi-main` or `asyncapi-bundle`

Reply:
also, with all other commands except of `generator` that has technical limitation, we should by default output to terminal, if `-o` is not provided